biome.json

```diff
@@ -5,6 +5,14 @@
   },
   "formatter": {
     "include": ["**/*.ts", "**/*.tsx"],
+    "ignore": [
+      "docs/.vitepress",
+      "node_modules",
+      ".next",
+      "packages/components/icons",
+      "**/*.(svg,js)",
+      "**/*/firestore.ts"
+    ],
```

Comment:
I don't know if we have a way to share these ignore statements across `formatter` and `linter` configs